pleo-io/actions
# Test version of Deployment notifications action

Action to send a message to PR when that PR's change has been deployed. 

### How it works
On a deployment:
1. get prev deployment’s sha https://docs.github.com/en/rest/reference/repos#list-deployments
2. get a list of commits since that sha https://docs.github.com/en/rest/reference/repos#list-commits
3. for each commit find a PR - https://docs.github.com/en/rest/reference/repos#list-pull-requests-associated-with-a-commit
4. send a message to that PR

### Test & Release plan

1. Run this action manually and check the logs;
2. have a separate PR to enable message send out.
3. add the workflow after `Deploy-app` on `product-web`
